cli/program_state.rs

```diff
@@ -137,6 +137,72 @@ impl ProgramState {
     Ok(Arc::new(program_state))
   }
 
+  /// Prepares a set of module specifiers for loading in one shot.
+  ///
+  pub async fn prepare_module_graph(
```

Comment:
Can this function could be reused for other purposes besides testing?

Reply:
Should be usable for `cache_subcommand` as-well.